scripts/constants.py

```diff
@@ -82,6 +82,7 @@
 COUNTDOWN_IMAGE_MAX_HEIGHT_RATIO = 0.2 #(0. - 1.) Ratio of the countdown images over screen height
 
 EMAIL_BUTTON_IMG  = os.path.join("ressources","ic_email.png")
+PRINT_BUTTON_IMG  = os.path.join("ressources","ic_print.png")
```

Comment:
Did you include `ic_print.png` in this pull request?

Reply:
I was using my own icon's at the time. I have a new icon to match the current still in the last commit.